editor/Dockerfile

```diff
@@ -119,22 +119,39 @@ RUN [ `echo $version-$module | grep -v '^\(2018.3\|2018.4\).*-android'` ] && exi
     echo ''; \
     echo 'xvfb-run -ae /dev/stdout "$UNITY_PATH/Editor/Unity" -batchmode "$@"'; \
   } > /usr/bin/unity-editor \
-  && chmod +x /usr/bin/unity-editor
+  && chmod +x /usr/bin/unity-editor \
+  && { \
+    echo "export ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT}"; \
+    echo "export ANDROID_HOME=${ANDROID_HOME}"; \
+    echo "export ANDROID_NDK_HOME=${ANDROID_NDK_HOME}"; \
+    echo "export JAVA_HOME=${JAVA_HOME}"; \
+    echo "export PATH=${PATH}"; \
+  } > ~/.bashrc
 
 #=======================================================================================
 # [2019.x/2020.x/2021.x-android] Support usage of installed Android SDKs outside of editor
 #=======================================================================================
-RUN [ `echo $version-$module | grep -v '^20(?!18).*-android'` ] && exit 0 || : \
+RUN [ `echo $version-$module | grep -vP '^20(?!18).*-android'` ] && exit 0 || : \
+  \
+  # Install jq
+  && apt-get update -qq \
+  && apt-get install -qq -y --no-install-recommends jq \
   \
   # Environment Variables
   && export RAW_ANDROID_SDK_ROOT=$(jq -cr '(.[] | select(.id == "android-sdk-platform-tools")).destination' $UNITY_PATH/modules.json) \
-  && export ANDROID_SDK_ROOT=${RAW_ANDROID_SDK_ROOT/'{UNITY_PATH}'/$UNITY_PATH} \
+  && export ESCAPED_UNITY_PATH=$(printf '%s' "$UNITY_PATH" | sed 's/[#\/]/\\\0/g') \
+  && export ANDROID_SDK_ROOT=$(echo $RAW_ANDROID_SDK_ROOT | sed -e "s/{UNITY_PATH}/$ESCAPED_UNITY_PATH/g") \
+  && export ANDROID_HOME=${ANDROID_SDK_ROOT} \
   && export RAW_ANDROID_NDK_ROOT=$(jq -cr '(.[] | select(.id == "android-ndk")).destination' $UNITY_PATH/modules.json) \
-  && export ANDROID_NDK_ROOT=${RAW_ANDROID_NDK_ROOT/'{UNITY_PATH}'/$UNITY_PATH} \
+  && export ANDROID_NDK_HOME=$(echo $RAW_ANDROID_NDK_ROOT | sed -e "s/{UNITY_PATH}/$ESCAPED_UNITY_PATH/g") \
```

Comment:
More consistent with 2018.x block

Reply:
In 2018 block it looks like this

```
&& export ANDROID_NDK_HOME=${ANDROID_SDK_ROOT}/ndk/${ANDROID_NDK_VERSION} \
```

Did you forget to add the ndk/{version} part or is this correct? (this is hard to check for me)

Reply:
I think this is why the variable was called NDK_ROOT and not NDK_HOME, although I'm not sure.

@mob-sakai could you please have a look as well?

Reply:
When installing the NDK with the sdkmanager, it places it into a versioned folder so that you can easily have multiple versions installed. Unity simply installs a single version which it places into a generic `NDK` folder.